packages/wrangler/src/index.tsx

```diff
@@ -1165,6 +1165,97 @@ function createCLIParser(argv: string[]) {
 		}
 	);
 
+	wrangler.command(
+		"secret:bulk [json]",
+		"🗄️ Bulk upload secrets for a Worker",
+		(yargs) => {
+			return yargs
+				.positional("json", {
+					describe: `The JSON file of key-value pairs to upload, in form {"key": value, ...}`,
+					type: "string",
+				})
+				.option("name", {
+					describe: "Name of the Worker",
+					type: "string",
+					requiresArg: true,
+				})
+				.option("env", {
+					type: "string",
+					requiresArg: true,
+					describe:
+						"Binds the secret to the Worker of the specific environment.",
+					alias: "e",
+				});
+		},
+		async (secretBulkArgs) => {
+			await printWranglerBanner();
+			const config = readConfig(
+				secretBulkArgs.config as ConfigPath,
+				secretBulkArgs
+			);
+
+			const scriptName = getLegacyScriptName(secretBulkArgs, config);
+			if (!scriptName) {
+				throw new Error(
+					"Required Worker name missing. Please specify the Worker name in wrangler.toml, or pass it as an argument with `--name <worker-name>`"
+				);
+			}
+
+			const accountId = await requireAuth(config);
+
+			if (secretBulkArgs.json) {
```

Comment:
If we use a required positional argument, then this will always be true and yargs will handle if the user does not give a json file.

Reply:
Good point, I thought I had removed this when I loved it into it's own command.